src/components/Tool/FixEmailAddresses/FixEmailAddressPerson.test.tsx

```diff
@@ -3,57 +3,72 @@ import { ThemeProvider } from '@mui/material/styles';
 import userEvent from '@testing-library/user-event';
 import { DateTime } from 'luxon';
 import TestWrapper from '__tests__/util/TestWrapper';
-import { render } from '__tests__/util/testingLibraryReactMock';
+import { render, waitFor } from '__tests__/util/testingLibraryReactMock';
 import theme from '../../../theme';
-import { FixEmailAddressPerson } from './FixEmailAddressPerson';
+import {
+  FixEmailAddressPerson,
+  FixEmailAddressPersonProps,
+} from './FixEmailAddressPerson';
+import { EmailAddressData } from './FixEmailAddresses';
 
 const testData = {
   name: 'Test Contact',
-  id: 'testid',
+  personId: 'testid',
   contactId: 'contactTestId',
-  emails: [
+  emailAddresses: [
     {
       source: 'DonorHub',
       updatedAt: DateTime.fromISO('2021-06-21').toString(),
       email: 'test1@test1.com',
       primary: true,
-    },
+      isValid: false,
+      personId: 'testid',
+      isPrimary: true,
+    } as EmailAddressData,
     {
       source: 'MPDX',
       updatedAt: DateTime.fromISO('2021-06-22').toString(),
       email: 'test2@test1.com',
       primary: false,
-    },
+      isValid: false,
+      personId: 'testid',
+      isPrimary: false,
+    } as EmailAddressData,
```

Comment:
It looks like this will be difficult to test since `isValid` and `personId` are not used on the page and `isPrimary` is set from `email.primary` without a default value. `emails` is not exposed to grab directly and the properties with default values are not on the DOM.